README.md

```diff
@@ -14,6 +14,8 @@ clipboard.
 
 A systemd user service for starting clipmenud is included at
 [init/clipmenud.service](https://github.com/cdown/clipmenu/blob/develop/init/clipmenud.service).
+It's installed as part of the manual install process below.
```

Comment:
I'd rather not recommend manual installation upfront, people should be using packages where possible.

Reply:
Ubuntu is the most popular desktop Linux distribution and there is no package for it. I was trying to provide easy to find instructions that would help people who are using the most popular Linux distro. How you do recommend documenting the install process for Ubuntu users?

Reply:
The way the `systemd` file is linked to directly here it's not clear that it's included in the packages (It is for Arch) or would otherwise be included during install. It seems like the suggestion is that the file might need to be copy/pasted or manually installed. 

I don't think people need to see the contents of the `systemd` file any more than they need to see the content of the `bash` scripts. Why not just remove the link and say:

> A systemd user service for starting clipmenud is installed as part of the project.

And just leave it that.

Reply:
On the other hand, all users who want to use the `systemd` integration need to actually enable and start the systemd service. I included that in the "manual install" section put perhaps it should be moved up to the "Usage" section, since it applies to those using packages as well. At least Arch doesn't actually `start` or `enable` the service for you.

Reply:
Different distributions have different ideas about what the right way to do things is. Arch just happens to (correctly...) have the idea that one shouldn't enable or start things up front. I'm pretty sure Debian and Ubuntu at least historically have taken the opposite perspective, so it's more of a distribution question than a package one.

> I don't think people need to see the contents of the systemd file any more than they need to see the content of the bash scripts.

You're going to probably end up looking at it if you want to customise any of the environment. I mean sure, you can use a dropin, but you're going to go and look at what's there already to work out what to do.